src/API/Site/Controllers/MerchantCenter/ShippingRateBatchController.php

```diff
@@ -85,6 +112,39 @@ protected function get_batch_create_callback(): callable {
 		};
 	}
 
+	/**
+	 * Get the callback for creating items via batch.
+	 *
+	 * @return callable
+	 */
+	protected function get_batch_suggestions_callback(): callable {
+		return function( Request $request ) {
+			$country_codes = $request->get_param( 'country_codes' );
```

Comment:
What happens if 'country_codes' is empty, should the answer still be successful or should we respond with an error  ?

Reply:
WordPress returns an `HTTP 400` error response if the required param is empty so no need for us to check manually. We do check the format through `get_country_code_validate_callback`.